pkg/resolution/resource/crd_resource_test.go

```diff
@@ -0,0 +1,337 @@
+/*
+Copyright 2023 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
```

Comment:
can you use package "resource_test" here? https://github.com/tektoncd/community/blob/main/standards.md#go-packages

Reply:
Yes, thank you very much for the link to the document.